languages/ga.ini

```diff
@@ -380,7 +380,7 @@ Empty Book Bag = "Folmhaigh an mála leabhar"
 empty_search_disallowed = "Ní cheadaítear iarratas folamh a dhéanamh i gcás an sprioc-chuardaigh reatha"
 Enable Auto Config = "Cumasaigh an t-uathchumrú"
 End Page = "Deireadh an leathanaigh"
-eol_ellipsis = "Ö"
+eol_ellipsis = "…"
```

Comment:
Do we need `eol_ellipsis` in the other language files?

Reply:
It should already be there in most. For those where it is currently undefined, VuFind will fall back to the en.ini version until translation volunteers provide a localized version.

Reply:
(You're only seeing diffs related to this string in this PR because the existing translation in ga.ini was incorrect and had to be fixed).